components/contentful-visual.coffee

```diff
@@ -70,7 +71,7 @@ export default
 				maxWidth
 
 				# Accessibility
-				alt: image?.description ||
+				alt: if nullAlt then "" else image?.description ||
```

Comment:
I think [this convention from `craft-visual`](https://github.com/BKWLD/cloak/blob/065fe1728565201ea68e4a0df39e11cb128536fb/components/craft-visual.coffee#L80) will work here:

```suggestion
				alt: props.alt ? image?.description ||
```

This says "if props.alt has a value, use it, otherwise, use one of the following choices".  The, in your project, you would set alt to an empty string.  Like:

```vue
<contentful-visual :image='image' alt='' />
```

Reply:
I think I understand it. Since the decorative images that have been added to HFO's contentful space follow the pattern of having the `description` field set to the string 'null', in this scenario to render a decorative image with the proper empty `alt` attribute we would write something like `<contentful-visual :image='image' :alt="image.description === 'null' ? '' : undefined">`, right?

Reply:
Oh, I see, in HFO we're actually storing the description as `"null"` ... huh.

Yeah, you got it.  However, if this behavior you want have globally, you _could_ make your own functional component that implements this.  For instance, you could do this, in the project:

```coffee
# ~/components/globals/contentful-visual.vue
import ContentfulVisual from '@bkwld/cloak/components/contentful-visual'
export default 

  render: (create, { data, props }) ->
    create ContentfulVisual, {
      ...data
      props: {
        ...props
        alt: if props.image?.description == 'null' then '' else props.alt
      }
    }
```

This should work because [we've setup](https://github.com/BKWLD/cloak/blob/master/modules/component-registration.js) the Cloak components to auto-import at a higher "level" so they can be overridden per-project if you make a component whose name resolves to the same thing (ie `contentful-visual`)

Reply:
Nice, I'll do that for both `contentful-visual` and `responsive-contentful-visual`. Thanks for the snippet and the info.